armi/reactor/grids/__init__.py

```diff
@@ -62,6 +62,7 @@
 """
 from typing import Tuple, Optional
 
+# ruff: noqa: F401
```

Comment:
```suggestion
# noqa: F401
```
I think the syntax is this?

Reply:
Actually, adding the prefix `ruff` makes the rule apply to the entire file.

And this file has about a dozen imports that aren't used in the FILE, but are part of the API.  So, we want to keep them.

I just think this is cleaner than having a dozen lines end with a `noqa` comment.